conda/recipes/libkvikio/meta.yaml

```diff
@@ -37,24 +37,60 @@ build:
 
 requirements:
   build:
-    - cmake >=3.20.1,!=3.25.0
-    - make
+    - cmake {{ cmake_version }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }} {{ cuda_version }}
+    - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - cudatoolkit {{ cuda_version }}.*
-  run:
-    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
+    - libcufile {{ libcufile_host_version }} # [linux64]
+    - libcufile-dev {{ libcufile_host_version }} # [linux64]
 
-test:
-  commands:
-    - test -f $PREFIX/include/kvikio/file_handle.hpp
-
-about:
-  home: http://rapids.ai/
-  license: Apache-2.0
-  license_family: Apache
-  license_file: LICENSE
-  summary: libkvikio library
+outputs:
+  - name: libkvikio
+    version: {{ version }}
+    script: install_libkvikio.sh
+    build:
+      number: {{ GIT_DESCRIBE_NUMBER }}
+      string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+      ignore_run_exports_from:
+        - {{ compiler('cuda') }}
```

Comment:
```suggestion
```

Since this dependency isn't listed as a `build` or `host` dependency of the `libkvikio` outputs package, we can remove the `ignore_run_exports_from` entry.